cmd/zqd/listen/command.go

```diff
@@ -152,6 +156,9 @@ func (c *Command) init() error {
 	if err := c.initZeek(); err != nil {
 		return err
 	}
+	if err := c.initWorkers(); err != nil {
+		return err
+	}
```

Comment:
Put this either before initZeek and initSurcata or after them, not between them.